app/controllers/api/v1/comments_controller.rb

```diff
@@ -0,0 +1,26 @@
+class Api::V1::CommentsController < ApplicationController
+    before_action :authenticate_user!, :except => [:show, :index]
+    
+    def index
+        @comments = Comment.all
+        render json: @comments
+    end
+
+    def show
```

Comment:
This probably isn't needed, since there probably isn't a use case for viewing a single comment.